httpcore5/src/main/java/org/apache/hc/core5/http/impl/bootstrap/HttpRequester.java

```diff
@@ -82,9 +82,14 @@
 import org.apache.hc.core5.pool.PoolEntry;
 import org.apache.hc.core5.pool.PoolStats;
 import org.apache.hc.core5.util.Args;
+import org.apache.hc.core5.util.ReflectionUtils;
 import org.apache.hc.core5.util.TimeValue;
 import org.apache.hc.core5.util.Timeout;
 
+import static org.apache.hc.core5.reactor.SingleCoreIOReactor.TCP_KEEPCOUNT;
```

Comment:
@kkewwei This should be avoided. Now classic transport services are directly coupled to a async transport class